src/api/routes/contact.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+
+const ContactFormController = require("../controllers/ContactFormController");
+
+router.post("/", ContactFormController.contact);
```

Comment:
Change this to /submit to avoid any conflicts with the front-end endpoint (which will most likely be /contact/ as well).